.github/workflows/performance.yml

```diff
@@ -0,0 +1,79 @@
+name: "Compare performance"
+
+on:
+  workflow_call:
+
+env:
+  GIT_CONFIG_GLOBAL: "/root/.gitconfig" # fix path in container (https://github.com/actions/runner/issues/2033)
+
+jobs:
+  compare-performance:
+    if: ${{ github.event_name == 'pull_request' }}
+    runs-on: "ubuntu-latest"
+    container:
+      image: "ghcr.io/tarantool/sdvg-ci:0.0.2"
+    permissions:
+      pull-requests: write
+      packages: read
+
+    steps:
+      - uses: "actions/checkout@v4"
+
+      - name: "Install requirements"
+        run: "make bench/prepare"
+
+      - name: "Try fetch cached benchmark result from master branch"
+        id: "cache-benchmark"
+        uses: "actions/cache/restore@v4"
+        with:
+          key: "benchmark-master"
+          path: "benchmark-master.txt"
+
+      - name: "Fetch master branch"
+        if: ${{ steps.cache-benchmark.outputs.cache-hit != 'true' }}
+        uses: "actions/checkout@v4"
+        with:
+          ref: ${{ github.event.repository.default_branch }}
+          path: "master_branch"
+
+      - name: "Create master benchmark"
+        if: ${{ steps.cache-benchmark.outputs.cache-hit != 'true' }}
+        working-directory: "master_branch"
+        run: make test/performance > ../benchmark-master.txt
+
+      - name: "Save benchmark-master.txt to cache"
+        if: ${{ steps.cache-benchmark.outputs.cache-hit != 'true' }}
+        uses: "actions/cache/save@v4"
+        with:
+          key: "benchmark-master"
+          path: "benchmark-master.txt"
+
+      - name: "Run benchmarks on PR branch"
+        run: make test/performance > benchmark-pr.txt
+
+      - name: "Compare results"
+        run: make bench/compare > performance-report.md
+
+      - uses: "mshick/add-pr-comment@v2"
+        with:
+          message-path: "performance-report.md"
+
+  update-performance-cache:
+    if: ${{ github.event_name == 'push' && github.ref_name == github.event.repository.default_branch }}
```

Comment:
I think it's better to remove the event name check so that you can manually run tests on master to refresh metrics

Reply:
fixed